packages/deployment-server-ui/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "deployment-server-ui",
+  "version": "0.0.0",
+  "description": "> TODO: description",
+  "author": "WoH <wolfgang@hey.com>",
```

Comment:
Should not set authors of packages we all work on to individual authors. This may be changed at a later stage though.